tests/stage-test/stage-key/staging_output.txt

```diff
@@ -1,5 +1,6 @@
 
   CEPH_VERSION      : luminous
+  CEPH_POINT_RELEASE: 
```

Comment:
Best to specify the CEPH_VERSION as `luminous-1.2.3-1` or something so we test that point releases are working properly in staging.

Reply:
Good idea.